src/aspire/nfft/finufft.py

```diff
@@ -1,33 +1,95 @@
-import finufftpy
+import logging
 import numpy as np
-
+import finufftpy
 from aspire.nfft import Plan
 from aspire.utils import ensure
+from aspire.utils.misc import complex_type
+
+
+logger = logging.getLogger(__name__)
 
 
 class FINufftPlan(Plan):
-    def __init__(self, sz, fourier_pts, epsilon=1e-15, **kwargs):
+    def __init__(self, sz, fourier_pts, epsilon=1e-15, many=0, **kwargs):
         """
         A plan for non-uniform FFT (3D)
+
         :param sz: A tuple indicating the geometry of the signal
         :param fourier_pts: The points in Fourier space where the Fourier transform is to be calculated,
             arranged as a 3-by-K array. These need to be in the range [-pi, pi] in each dimension.
```

Comment:
Need to change the "3-by-K array" for cases of different dimensions.

Reply:
The circle with `Outdated` means you are looking at old code.  I went ahead and changed those after you found in cufinufft.py. Try refereshing before you get too far.